README.md

```diff
@@ -1,25 +1,77 @@
 # 3scale-operator
 
+### Project Status: alpha
+
+The project is currently alpha which means that there are still new feautres and APIs planned that will be added in the future.
+Dueo to this breaking changes may still happen.
+
+Only use for short-lived testing clusters. Do not deploy in a project with a 3scale API Management solution.
+
+## Overview
+
+This project contains the 3scale operator software. 3scale operator is a
+software based on [Kubernetes operators](https://coreos.com/operators/) that
+provides:
+* A way to install a 3scale API Management solution, providing configurability
+  options at the time of installation
+* Ability to define 3scale API definitions and set
+  them into a 3scale API Management solution
+
 ## Prerequisites
 
 - [operator-sdk] version v0.2.1.
 - [dep][dep_tool] version v0.5.0+.
 - [git][git_tool]
 - Access to a Openshift v3.11.0+ cluster.
 
-## Quick Start
+## Getting started
 
-To download and prepare the environment for 3scale-operator:
+Download the 3scale-operator project:
 
 ```sh
-$ mkdir -p $GOPATH/src/github.com/3scale
-$ cd $GOPATH/src/github.com/3scale
-$ git clone https://github.com/3scale/3scale-operator
-$ cd 3scale-operator
-$ git checkout master
-$ make vendor
+mkdir -p $GOPATH/src/github.com/3scale
+cd $GOPATH/src/github.com/3scale
+git clone https://github.com/3scale/3scale-operator
+cd 3scale-operator
+git checkout master
 ```
 
+To see an example deploy of the 3scale-operator and its CRDs refer
+to the following [User guide](doc/user-guide.md)
+
+## Development
+
+Download the 3scale-operator project and install the needed dependencies:
+
+```sh
+mkdir -p $GOPATH/src/github.com/3scale
```

Comment:
This is the same as above. Should not be needed to repeat.

Reply:
There was one additional step. I modified the content to refer to the other section from here and just add the additional step